packages/react/package.json

```diff
@@ -26,7 +26,7 @@
     "./{cjs,esm}/*/index.js"
   ],
   "dependencies": {
-    "@figspec/components": "^0.1.1"
+    "@figspec/components": "^0.1.7"
```

Comment:
The React binding is compatible with `>= v0.1.1` so no need to change here.

Reply:
Sure thing!